src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -205,6 +208,18 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
     enabled: queryEnabled,
   })
 
+  const supportsMultiAccount = useMemo(() => {
+    if (!wallet || !accounts) return false
+    if (!isMetaMaskMultichainWallet) return true
+
+    return canAddMetaMaskAccount({
+      accountNumber: accounts.pages.length,
+      chainId,
+      wallet,
+      isSnapInstalled: !!isSnapInstalled,
+    })
+  }, [chainId, wallet, accounts, isMetaMaskMultichainWallet, isSnapInstalled])
+
```

Comment:
We still need to check for BIP44 feature capability which is the source of truth - MM is the odd one because of the snap un/installed variant, but we never want to get into a state where we detect a wallet that doesn't set its BIP44 multi-accounts flag to true as multi-account.

```suggestion
 const supportsMultiAccount = useMemo(() => {
  // No hdwallet BIP44 multi-account feature capability in hdwallet, we don't support multi-account
  if (!wallet?.supportsBip44Accounts()) return false
  // MetaMask is the odd one with the snap installed/uninstalled variant - all others wallets honor their BIP44 multi-account support flag
  if (!isMetaMaskMultichainWallet) return true

  return canAddMetaMaskAccount({
    accountNumber: accounts.pages.length,
    chainId,
    wallet,
    isSnapInstalled: !!isSnapInstalled,
  })

  return true
}, [chainId, isSnapInstalled, wallet])

```

Reply:
It's been pushed!